tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/containers/BrokerContainer.java

```diff
@@ -29,4 +34,26 @@ public BrokerContainer(String clusterName, String hostName) {
         super(
             clusterName, hostName, hostName, "bin/run-broker.sh", BROKER_PORT, INVALID_PORT);
     }
+
+    public void stopBroker() {
+        try {
+            this.execCmd("supervisorctl", "stop", "broker");
+        } catch (Exception ex){
+
+        }
+    }
+
+    public void startBroker() {
+        try {
+            this.execCmd("supervisorctl", "start", "broker");
+        } catch (Exception ex){
+
+        }
+
+        PulsarCluster.waitForPort(60,
```

Comment:
why can't you reuse testcontainers WaitStrategy, rather than reinventing the wheels? https://github.com/testcontainers/testcontainers-java/blob/master/core/src/main/java/org/testcontainers/containers/wait/strategy/WaitStrategy.java

new HostPortWaitStrategy().waitUntilReady(this)